pkgs/applications/video/kaiscr/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitea
+, gobject-introspection
+, gtk3
+}:
+
+buildPythonApplication rec {
+  pname = "KaiScr";
+  version = "unstable-2020-10-21";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "farooqkz";
+    repo = pname;
+    rev = "3ebb16d29075d09fe0bb69f5a00d57e107cae6c2";
+    sha256 = "sha256-JnDjExGCfDKCvgH+h5QwYaTzaQ+2YEAkiSFrlw9PGOc=";
+  };
+
+  # src is plain Python3 scripts.
+  format = "other";
+  dontUseSetuptoolsBuild = true;
+  dontUsePipInstall = true;
+  dontUseSetuptoolsCheck = true;
+
+  postPatch = ''
+    # kailive isn't marked as exec
+    chmod +x kailive.py
+  '';
+
+  buildInputs = [
+    gobject-introspection
+    gtk3
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp kailive.py $out/bin/kailive
+    cp kaiscr.py $out/bin/kaiscr
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    install -Dm755 kailive.py $out/bin/kailive
    install -Dm755 kaiscr.py $out/bin/kaiscr
    runHook postInstall
  '';
```

Also, I'm not sure that this will generate a proper wrapping around these scripts.

---

It did for `kaiscr`. Forgot to add a shebang to `kailive` so that's not working. Good catch ^^